llvm/test/CodeGen/SPIRV/hlsl-intrinsics/WaveActiveAnyTrue.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv32v1.3-vulkan-unknown %s -o - | FileCheck %s
```

Comment:
I think we prefer `-mtriple=spirv-unknown-unknown`. You might need to use `spirv1.[5|6]` if you are using an new included spirv op.

Reply:
do you mean `-mtriple=spirv32v1.5-unknown-unknown`? I couldn't get any other variant of the `spirv` part to work